src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -193,6 +193,12 @@
   <data name="DeleteDialogAreYouSure.Text" xml:space="preserve">
     <value>Are you sure you want to delete this entry?</value>
   </data>
+  <data name="DeleteEntryBtn.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Delete</value>
+  </data>
+  <data name="DeleteEntryBtn.[using:Microsoft.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Delete (Delete)</value>
```

Comment:
I think this entry needs a comment to mention that the Delete between parentheses refers to the Delete keyboard key.

Reply:
Commend added